packages/react/package.json

```diff
@@ -67,17 +67,20 @@
   "devDependencies": {
     "@arethetypeswrong/cli": "^0.15.4",
     "@types/node": "^20.14.12",
-    "@types/react": "^18.3.3",
-    "@types/react-dom": "^18.3.0",
-    "react": "^18.3.1",
-    "react-dom": "^18.3.1",
+    "@types/react": "*",
+    "@types/react-dom": "*",
     "tsup": "^8.2.1",
     "typescript": "4.9.5"
   },
   "peerDependencies": {
     "react": ">=17",
     "react-dom": ">=17"
   },
+  "peerDependenciesMeta": {
+    "react-dom": {
+      "optional": true
+    }
+  },
```

Comment:
Marking the `react-dom` is an optional peer dependency to support using this package in the RN projects.